### ISSUE TITLE
Add an awesome Cadet - Project management / Github Issues board

### ISSUE THREAD

**Issue by ei-grad** [isRuslan/awesome-elm#218]
https://github.com/vito/cadet - Project management / Github issues board written by @vito and [used by Concourse CI team](https://project.concourse-ci.org/)

Probably this could go to examples?

**Comment by sporto** [comment]
Yes, I'm happy to put this on examples, if you are keen to make a PR. Thanks
